### DISCUSSION TITLE
Leaflet v2.0

### DISCUSSION THREAD

**Discussion by jmuinos** [Leaflet/Leaflet#9383] [Q&A]
Hi!  
I'm currently working on an enormous project and I've been asked to renew every geolocation tool on the app and I've been working with Leaflet for 2 years, so I would like to use it for this project too.

My questions are:

- Is there already a release date for v2.0?
- Does Leaflet v2.0 implement clustering features natively or will we depend on plugins as in previous versions?

Thank you in advance,

**Слава Україні!**

**Comment by Falke-Design** [comment]
Hi @jmuinos,

> Is there already a release date for v2.0?

we don't have a release date currently. We have still some open tasks to resolve and we are always searching for help.

> Does Leaflet v2.0 implement clustering features natively or will we depend on plugins as in previous versions?

This feature will still depend on plugins.
